### ISSUE TITLE
Help me stop martingale.

### ISSUE THREAD

**Issue by thawatchaii** [binary-com/binary-bot#1451]
If this bot martingale to stake = 20 then lose, please stop it then reset to the start.
[-1.25.zip](https://github.com/binary-com/binary-bot/files/3119617/-1.25.zip)

**Comment by hotmatrixx** [comment]
Your bot isn't actually working.

It's missing at least the Martingale multiplier, and the notification/report section, possibly the calculator that you're using to adjust the martingale - if you have one.

So I'm asking;
have you removed them, or are you using someone else's script and not quite sure how it works? there are chunks missing. are you aware of that?

The stuff I have added is presuming you removed them on purpose. If you haven't, then I guess you're aware that there is no martingale calculator ATM so my additions would be 'incomplete'.

It's pretty clear that you pulled the guts out of it.... If you want people to help you for free, then it might be nice to share your dieas with them too.

Regardless.
I've added little text bubbles to show you the bits I added.
[1.26.zip](https://github.com/binary-com/binary-bot/files/3119812/1.26.zip)


hope it helps yo uon your road to riches.
